pkg/resources/awsStrategyHelper.go

```diff
@@ -1,738 +0,0 @@
-package resources
-
-import (
-	"context"
-	"encoding/json"
-	"errors"
-	"fmt"
-
-	l "github.com/integr8ly/integreatly-operator/pkg/resources/logger"
-
-	"github.com/aws/aws-sdk-go/service/elasticache"
-
-	"github.com/aws/aws-sdk-go/service/rds"
-
-	croUtil "github.com/integr8ly/cloud-resource-operator/pkg/client"
-
-	awsSdk "github.com/aws/aws-sdk-go/aws"
-	"github.com/integr8ly/cloud-resource-operator/pkg/providers/aws"
-	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
-
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
-	corev1 "k8s.io/api/core/v1"
-)
-
-const (
-	configMapName    = "cloud-resources-aws-strategies"
-	PostgresStratKey = "postgres"
-	RedisStratKey    = "redis"
-)
-
-// At the moment this function only allows for overriding the createStrategy. It's unlikely that delete or region will need to be
-// overridden. In the case that they do, the function can be extended.
-func CreatePostgresTierFromProduction(ctx context.Context, client k8sclient.Client, ns string, useClusterStorage bool, tier string, createStrategy map[string]interface{}, logger l.Logger) error {
```

Comment:
These are unused inside the project. there are some more specific helpers inside the CRO project which should probably be used instead. If necessary I can move this change to another pr as it's not essential for the bug fix.